static/js/load_map.js

```diff
@@ -75,4 +76,60 @@ function initialize() {
             markers.push(nextMarker);
         });
     });
+
+    const stores = document.getElementById('Stores');
+    function getLocation() {
+        if (navigator.geolocation) {
+            return navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            x.innerHTML = "Geolocation is not supported by this browser.";
```

Comment:
Yeah, should probably make that consistent -- we seem to be using different indentations so I'll add a `.jshintrc`.